### PR TITLE
added solver version to examples for automated meshing

### PR DESCRIPTION
**Pull request by maciej-flexcompute**

### CODE REVIEW

File: docs/source/preprocessing/automatedMeshing.rst [hunk 1]
```diff
@@ -92,7 +96,7 @@ The volume mesh is created by :code:`NewMeshFromSurface()` function using :code:
 
 .. code-block:: python
 
-    volumeMeshId = flow360client.NewMeshFromSurface(surfaceMeshId, "volumeMesh.json", meshName="my_volume_mesh")
+    volumeMeshId = flow360client.NewMeshFromSurface(surfaceMeshId, "volumeMesh.json", meshName="my_volume_mesh",  solverVersion='release-21.4.1.0')
```
Comment (lei-flex) [inline docs/source/preprocessing/automatedMeshing.rst:99]: the api has default version assigned already.

File: docs/source/preprocessing/automatedMeshing.rst [hunk 2]
```diff
@@ -117,6 +121,9 @@ Outputs\:
 
 For a full description of config.json for volume mesher see :ref:`here.<JSON volume mesher>`
 
+.. note::
```
Comment (lei-flex) [inline docs/source/preprocessing/automatedMeshing.rst:124]: the missing part is from Geo to surface.

File: docs/source/preprocessing/automatedMeshing.rst [hunk 3]
```diff
@@ -48,7 +48,7 @@ The surface mesh is created by submitting a geometry file and JSON file using :c
 .. code-block:: python
 
     import flow360client
-    surfaceMeshId = flow360client.NewSurfaceMeshFromGeometry("path/to/geometry.csm", "surfaceMesh.json", surfaceMeshName="my_surface_mesh")
+    surfaceMeshId = flow360client.NewSurfaceMeshFromGeometry("path/to/geometry.csm", "surfaceMesh.json", surfaceMeshName="my_surface_mesh", solverVersion='release-21.4.1.0')
```
Comment (rundaji) [inline docs/source/preprocessing/automatedMeshing.rst:51]: We only need to input the solver version when calling NewSurfaceMeshFromGeometry()

File: docs/source/preprocessing/automatedMeshing.rst (hunk 1 above)
Comment (rundaji) [reply docs/source/preprocessing/automatedMeshing.rst:99]: No need to input solver version when calling NewMeshFromSurface()

**Review by rundaji — APPROVED** [review]

**Review by lei-flex — APPROVED** [review]

**Pull request merged**